tests/Test/AbstractFixerWithAliasedOptionsTestCase.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Test;
+
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\FixerConfiguration\AliasedFixerOption;
+
+/**
+ * @author ntzm
+ *
+ * @internal
+ */
+abstract class AbstractFixerWithAliasedOptionsTestCase extends AbstractFixerTestCase
+{
+    /**
+     * @var null|\PhpCsFixer\Fixer\ConfigurableFixerInterface
+     */
+    private $fixerWithAliasedConfig;
```

Comment:
Please override `tearDown()` or `tearDownAfterClass()` to unset this property after tests and avoid memory leaks.

Reply:
Yep, but this is assuming PHPUnit will stop referencing the instance of the test as soon as all of its test methods have been executed. In my experience this is rarely (never?) true. For that reason I avoid using properties in tests at all. And if I have to, I always use `tearDown` or `tearDownAfterClass` to unset them.

Reply:
Done 👍